book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -249,163 +542,29 @@ Only if this section is defined a sensitivity analysis is done. This section wil
 </sensitivity.analysis>
 ```
 
-* **quantiles** : [optional] Quantiles of parameter distributions at which the model should be evaluated when running sensitivity analysis. Values greater than 0 and less than 1 can be used.  
-* **sigma** : [optional] Any real number can be used to indicate the quantiles to be used in units of normal probability.  
-* **quantile** : [optional] Which quantile should be used.  
-* **start.date** : [required?] start date of the sensitivity analysis (in YYYY/MM/DD format) 
-* **end.date** : [required?] end date of the sensitivity analysis (in YYYY/MM/DD format)
+* `quantiles` : [optional] Quantiles of parameter distributions at which the model should be evaluated when running sensitivity analysis. Values greater than 0 and less than 1 can be used.
+* `sigma` : [optional] Any real number can be used to indicate the quantiles to be used in units of normal probability.
+* `quantile` : [optional] Which quantile should be used.
```

Comment:
I think the addition of this tag is a mistake (I've never heard of us having both a quantiles and a quantile tag

Reply:
So it's worth confirming, but I _think_ this may be set up such that `quantile` is for specifying specific quantiles within the `quantiles` block. In other words, you may be able to do either syntax like this...


```xml
<quantiles>
  <sigma>-1</sigma>
  <sigma>1</sigma>
</quantiles>
```

...or syntax like this...

```xml
<quantiles>
  <quantile>0.025</quantile>
  <quantile>0.975</quantile>
</quantiles>
```

...and possibly mixing and matching. But someone should check this. In 0d5453ef3, I dropped the `quantile` documentation because it doesn't seem to be set at all by the web workflow.